src/documentation/0009-node-environment-variables/index.md

```diff
@@ -1,12 +1,30 @@
 ---
 title: How to read environment variables from Node.js
 description: 'Learn how to read and make use of environment variables in a Node.js program'
-authors: flaviocopes, MylesBorins, fhemberger, LaRuaNa, ahmadawais
+authors: flaviocopes, MylesBorins, fhemberger, LaRuaNa, ahmadawais, aymen94
 section: Getting Started
 ---
 
 The `process` core module of Node.js provides the `env` property which hosts all the environment variables that were set at the moment the process was started.
 
+## We have two way to set variables:
+- using the command line
+- using a `.env` file
+
+### Using Command Line:
+
+`PORT=8626 node app.js`
+
+### Using a `.env` File:
+
+Create the .env file in the root of your app and add your variables and values to it.
```

Comment:
This will only work if using a module such as [`dotenv`](https://www.npmjs.com/package/dotenv).

Reply:
@richardlau  Add dotenv or not?